---
layout: default
title: {{ site.name }}
---

<div id="home">
  <h1>Posts</h1>
  <ul class="posts">
    {% for post in site.posts %}
      <li><span>{{ post.date | date_to_string }}</span> &raquo; <a href="{{ site.baseurl }}{{ post.url }}">{{ post.title }}</a></li>
    {% endfor %}
  </ul>
  <h1>Application List</h1>
  <ul class="posts">
      <li><span>Sequence Conservation</span></li>
        <ul class="posts">
            <li><a href="https://github.com/cyversewarwick/apples">APPLES</a> &raquo; A set of tools to analyse promoter sequences on a genome-wide scale. </li>
        </ul>
      <li><span>Time Series Expression Data</span></li>
        <ul class="posts">
            <li><span>Clustering / Biclustering</span></li>
              <ul class="posts">
                <li><a href="https://github.com/cyversewarwick/bhc">BHC</a> &raquo; A clustering algorithm for expression data originally made available in R, allows for the analysis of both time course or multiple static datasets.</li>
                <li><a href="https://github.com/cyversewarwick/tcap">TCAP</a> &raquo; A clustering algorithm for time course expression data, identifies complex regulatory groups thanks to a rich information measure.</li>
                <li><a href="https://github.com/cyversewarwick/wigwams">Wigwams</a> &raquo; An algorithm for the extraction of gene groups co-regulated across subsets of multiple time course datasets. </li>
              </ul>
            <li><span>Differential Expression</span></li>
              <ul class="posts">
                <li><a href="https://github.com/cyversewarwick/gp2s">GP2S</a> &raquo; A differential expression algorithm for time series data with a two condition (eg. control/treated) experimental design.</li>
                <li><a href="https://github.com/cyversewarwick/gradienttool">Gradient Tool</a> &raquo; An algorithm for the identification of the time of change from single condition time course expression data.</li>
              </ul>
            <li><span>Network Inference</span></li>
              <ul class="posts">
                <li><a href="https://github.com/cyversewarwick/csi">CSI</a> &raquo; A network inference algorithm capable of inferring causal regulatory network models from time course expression data.</li>
                <li><a href="https://github.com/cyversewarwick/hcsi">hCSI</a> &raquo; An expansion of CSI network inference to handle multiple time course datasets.</li>
                <li><a href="https://github.com/cyversewarwick/ocsi">oCSI</a> &raquo; An expansion of CSI network inference to handle data from multiple organisms.</li>
              </ul>
        </ul>
      <li><span>Enrichment Analysis</span></li>
        <ul class="posts">
          <li><span>Transcription Factor Binding Sites</span></li>
            <ul class="posts">
              <li><a href="https://github.com/cyversewarwick/hmt">HMT</a> &raquo; A transcription factor binding site overrepresentation analysis algorithm for known motifs.</li>
              <li><a href="https://github.com/cyversewarwick/hmt_index">HMT_index</a></li>
            </ul>
        </ul>
      <li><span>Genome-wide Chromatin Data</span></li>
        <ul class="posts">
          <li><span>Footprint Identification</span></li>
            <ul class="posts">
              <li><a href="https://github.com/cyversewarwick/wellington-footprint">Wellington Footprint</a> &raquo; An algorithm for the identification of regions occupied by proteins in DNase-seq data.</li>
              <li><a href="https://github.com/cyversewarwick/wellington-bootstrap">Wellington Bootstrap</a> &raquo; An algorithm for the identification of regions occupied by proteins in DNase-seq data, performing a differential analysis between two samples.</li>
            </ul>
        </ul>
  </ul>
  <div class="post">
  <p><a href="https://cyversewarwick.github.io/2017/02/14/application-access-guide.html">How do I run these applications?</a></p>
  </div>
</div>
