---
layout: post
title: "Related Publications"
date: 2016-07-13
---

|   Paper | Application |
|-------------|-------------:|
| Emily Breeze, Elizabeth Harrison, Stuart McHattie, Linda Hughes, Richard Hickman, Claire Hill, Steven Kiddle, Youn-sung Kim, Christopher A. Penfold, Dafyd Jenkins, Cunjin Zhang, Karl Morris, Carol Jenner, Stephen Jackson, Brian Thomas, Alexandra Tabrett, Roxane Legaie, Jonathan D. Moore, David L. Wild, Sascha Ott, David Rand, Jim Beynon, Katherine Denby, Andrew Mead, and Vicky Buchanan-Wollaston. **High-Resolution Temporal Profiling of Transcripts during Arabidopsis Leaf Senescence Reveals a Distinct Chronology of Processes and Regulation.** The Plant Cell, 2011. [doi:10.1105/tpc.111.083345](http://dx.doi.org/10.1105/tpc.111.083345) | [Gradient Tool](https://github.com/cyversewarwick/gradienttool) |
| Jason Piper, Markus C. Elze, Pierre Cauchy, Peter N. Cockerill, Constanze Bonifer, and Sascha Ott. **Wellington: A Novel Method for the Accurate Identification of Digital Genomic Footprints from DNase-Seq Data.** Nucleic Acids Research, 2013. [doi:10.1093/nar/gkt850](http://dx.doi.org/10.1093/nar/gkt850) | [Wellington-footprint](https://github.com/cyversewarwick/wellington-footprint/) |
| Jason Piper, Salam A Assi, Pierre Cauchy, Christophe Ladroue, Peter N Cockerill, Constanze Bonifer, and Sascha Ott. **Wellington-Bootstrap: Differential DNase-Seq Footprinting Identifies Cell-Type Determining Transcription Factors.** BMC Genomics, 2015. [doi:10.1186/s12864-015-2081-4](http://dx.doi.org/10.1186/s12864-015-2081-4) | [Wellington-bootstrap](https://github.com/cyversewarwick/wellington-bootstrap/) |
| Emily Breeze, Elizabeth Harrison, Stuart McHattie, Linda Hughes, Richard Hickman, Claire Hill, Steven Kiddle, Youn-sung Kim, Christopher A. Penfold, Dafyd Jenkins, Cunjin Zhang, Karl Morris, Carol Jenner, Stephen Jackson, Brian Thomas, Alexandra Tabrett, Roxane Legaie, Jonathan D. Moore, David L. Wild, Sascha Ott, David Rand, Jim Beynon, Katherine Denby, Andrew Mead, and Vicky Buchanan-Wollaston, **High-Resolution Temporal Profiling of Transcripts during Arabidopsis Leaf Senescence Reveals a Distinct Chronology of Processes and Regulation**, *Plant Cell*, 2011, [doi: 10.1105/tpc.111.083345](http://dx.doi.org/10.1105/tpc.111.083345) | [HMT](https://github.com/cyversewarwick/hmt/)|
| Nathaniel J. Davies, Peter Krusche, Eran Tauber and Sascha Ott, **Analysis of 5â€™ gene regions reveals extraordinary conservation of novel non-coding sequences in a wide range of animals**, *BMC Evolutionary Biology*, [doi: 10.1186/s12862-015-0499-6](http://dx.doi.org/10.1186/s12862-015-0499-6) | [APPLES](https://github.com/cyversewarwick/apples) - Conservation Tool |
| Laura Baxter, Aleksey Jironkin, Richard Hickman, Jay Moore, Christopher Barrington, Peter Krusche, Nigel P. Dyer, Vicky Buchanan-Wollaston, Alexander Tiskin, Jim Beynon, Katherine Denby, and Sascha Ott, **Conserved Noncoding Sequences Highlight Shared Components of Regulatory Networks in Dicotyledonous Plants**, *Plant Cell*, 2012, [doi:10.1105/tpc.112.103010](http://dx.doi.org/10.1105/tpc.112.103010) | [APPLES](https://github.com/cyversewarwick/apples) - Conservation Tool |
| Krzysztof Polanski, Johanna Rhodes, Claire Hill, Peijun Zhang, Dafyd J. Jenkins, Steven J. Kiddle, Aleksey Jironkin, Jim Beynon, Vicky Buchanan-Wollaston, Sascha Ott and Katherine J. Denby, **Wigwams: identifying gene modules co-regulated across multiple biological conditions**, *Bioinformatics*, 2014, [doi:10.1093/bioinformatics/btt728](http://bioinformatics.oxfordjournals.org/content/30/7/962.long) | [Wigwams](https://github.com/cyversewarwick/wigwams/) |
| Emma J Cooke, Richard S Savage, Paul DW Kirk, Robert Darkins and David L Wild, **Bayesian hierarchical clustering for microarray time series data with replicates and outlier measurements**, *BMC Bioinformatics*, 2011, [doi: 10.1186/1471-2105-12-399](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-399)  | [BHC](https://github.com/cyversewarwick/bhc/) |
| Christopher A. Penfold, David L. Wild, **How to infer gene networks from expression profiles, revisited**, *Interface Focus*, 2011, [doi: 10.1098/rsfs.2011.0053](http://rsfs.royalsocietypublishing.org/content/1/6/857.short)  | [CSI](https://github.com/cyversewarwick/csi/) |
|Christopher A. Penfold, Vicky Buchanan-Wollaston, Katherine J. Denby, and David L. Wild, **Nonparametric Bayesian inference for perturbed and orthologous gene regulatory networks**, *Bioinformatics*, 2012, [doi:10.1093/bioinformatics/bts222](http://dx.doi.org/10.1093/bioinformatics/bts222) | [hCSI](https://github.com/cyversewarwick/hcsi/) |
| Christopher A. Penfold, Jonathan B. A. Millar and David L. Wild, **Inferring orthologous gene regulatory networks using interspecies data fusion**, *Bioinformatics*, 2015, [10.1093/bioinformatics/btv267](http://dx.doi.org/10.1093/bioinformatics/btv267) | [OCSI](https://github.com/cyversewarwick/ocsi/) |
| Steven J. Kiddle, Oliver P. F. Windram, Stuart McHattie, Andrew Mead, Jim Beynon, Vicky Buchanan-Wollaston, Katherine J. Denby, and Sach Mukherjee, **Temporal clustering by affinity propagation reveals transcriptional modules in Arabidopsis thaliana**, *Bioinformatics*, 2010, [doi:10.1093/bioinformatics/btp673](https://bioinformatics.oxfordjournals.org/content/26/3/355.full) | [TCAP](https://github.com/cyversewarwick/tcap/) |
|Oliver Stegle, Katherine J. Denby, Emma J. Cooke, David L. Wild, Zoubin Ghahramani, and Karsten M. Borgwardt, **A Robust Bayesian Two-Sample Test for Detecting Intervals of Differential Gene Expression in Microarray Time Series**, *Journal of Computational Biology*, 2010. [doi:10.1089/cmb.2009.0175.](http://online.liebertpub.com/doi/abs/10.1089/cmb.2009.0175) | [GP2S](https://github.com/cyversewarwick/gp2s/) |
